app/assets/stylesheets/main.scss

```diff
@@ -5,6 +5,12 @@
 
 /* Dashboard */
 
+.link_disabled {
+  color: #000;
```

Comment:
I'm pretty sure there's a "black" variable in constants.scss. If not, you can make one.